src/utils/datatool/CMakeLists.txt

```diff
@@ -1,9 +1,22 @@
-set(datatool_SOURCES datatool.cpp)
+set(datatool_SOURCES datatool.cpp SceneLoader.cpp)
 
 add_executable(datatool ${datatool_SOURCES})
 
+# Assimp. Use a system package, if preferred.
+if(USE_SYSTEM_ASSIMP)
+  find_package(ASSIMP REQUIRED)
+else()
+  set(ASSIMP_BUILD_ASSIMP_TOOLS OFF CACHE BOOL "ASSIMP_BUILD_ASSIMP_TOOLS" FORCE)
+  set(ASSIMP_BUILD_TESTS OFF CACHE BOOL "ASSIMP_BUILD_TESTS" FORCE)
+  set(BUILD_SHARED_LIBS OFF CACHE BOOL "ASSIMP_BUILD_TESTS" FORCE)
+  add_subdirectory("${DEPS_DIR}/assimp" "${CMAKE_CURRENT_BINARY_DIR}/assimp")
+  include_directories(SYSTEM "${DEPS_DIR}/assimp/include")
+  # message(STATUS "Found ASSIMP in ${ASSIMP_INCLUDE_DIR}")
+endif()
```

Comment:
Why not moving just the `find_package()` here and keeping the rest inside `dependencies.cmake`? That should work even when Assimp is not present.

Also, I'm not sure about  `"${CMAKE_CURRENT_BINARY_DIR}/assimp"`, that feels out of place :)